src/AppProviders.tsx

```diff
@@ -73,8 +73,8 @@ export function AppProviders({ children }: ProvidersProps) {
                       <WalletConnectV2Provider>
                         <KeepKeyProvider>
                           <ErrorBoundary FallbackComponent={ErrorPage} onError={handleError}>
-                            <WagmiConfig config={wagmiConfig}>
-                              <QueryClientProvider>
+                            <QueryClientProvider>
```

Comment:
- `<QueryClientProvider />` first for wagmi *not* to import its own `useQuery` first and rug things - fine to revert if we want to as this is really just paranoia
- `<WagmiProvider />` over the deprecated `<WagmiConfig />`

Reply:
🧠 very nice